src/json-schema.ts

```diff
@@ -108,7 +108,7 @@ type PropertiesConstraint<
   : unknown;
 
 type AdditionalPropertiesConstraint<
-  AdditionalProperties extends SchemaLike | undefined
+  AdditionalProperties extends SchemaLike | (false | undefined)
```

Comment:
Recreating [my original comment from the other PR](https://github.com/ostrowr/ts-json-validator/pull/29#discussion_r377217202) here:

I didn't want to allow `additionalProperties: true`, so I'm specifically doing `(false | undefined)` instead of `(boolean | undefined)`